Mathlib/Algebra/Homology/Additive.lean

```diff
@@ -170,7 +170,7 @@ theorem NatTrans.mapHomologicalComplex_comp (c : ComplexShape ι) {F G H : W₁
       NatTrans.mapHomologicalComplex α c ≫ NatTrans.mapHomologicalComplex β c := by
   aesop_cat
 
-@[reassoc (attr := simp 1100)]
+@[reassoc]
```

Comment:
This removes the `simp` attribute entirely. Is that intentional?

Reply:
Oh, never mind, the diff hid the fact that the proof is `simp`.

Reply:
Yes, these are both already proved by `simp`.